draft-dekater-scion-dataplane.md

```diff
@@ -1462,6 +1466,7 @@ When an AS's forwarding key is compromised, an attacker can forge Hop Field MACs
 Unless an attacker has access to the forwarding keys of all ASes on the illegitimate path it wants to fabricate, it will need to splice fragments of two legitimate path segments with an illegitimate Hop Field. For this, it needs to create a Hop Field with a MAC that fits into the MAC chain expected by the second path segment fragment. The only input that the attacker can vary relatively freely is the 8-bit ``ExpTime``, but the resulting MAC needs to match a specific 16 bit ``Acc`` value. While there is a low probability of this working for a specific attempt (1/256), the attack will succeed eventually if the attacker can keep retrying over a longer time period or with many different path segment fragments.
 
 While a forwarding key compromise and the resulting loss of path authorization is a serious degradation of SCION's routing security properties, this does not affect access control or data security for the hosts in the affected AS. Unauthorized paths are available to the attacker, but the routing of packets from legitimate senders is not affected.
+Future work may tackle this with alternative forwarding key distribution approaches.
```

Comment:
Worth mentioning that a forwarding key can be rotated, immediately invalidating all path segments that still use the compromised key. I'm not sure alternative forwarding key distribution is necessary.